### ISSUE TITLE
Real company discount

### ISSUE THREAD

**Issue by tj** [apex/up#486]
It's already 50% off right now, but later I'd like to offer discounts for real companies and indie hackers as well. Startups don't need discounts sorry haha.

**Comment by kaihendry** [comment]
You shouldn't discount to companies imo. It makes decision makers nervous.

Oh btw, another thing I ran into with the accounts department is that I had to explain in layman's terms what Up does. In the end I think I scribbled they went with "software".

**Comment by tj** [comment]
hahah yea fair enough! I'll have to come up with a list of selling points for non-technical people higher up in the companies. Once it's more robust it should be a pretty easy sell if it's saving days or weeks of dev time
